Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -539,41 +503,41 @@ void CGPathAddEllipseInRect(CGMutablePathRef path, const CGAffineTransform* m, C
  @Status Interoperable
 */
 void CGPathCloseSubpath(CGMutablePathRef path) {
-    if (path == NULL) {
-        return;
+    // If we need to, close the path.
+    if (path->_startingPoint.x != path->_currentPoint.x || path->_startingPoint.y != path->_currentPoint.y) {
+        CGPathAddLineToPoint(path, nullptr, path->_startingPoint.x, path->_startingPoint.y);
     }
-
-    _CGPathAddElement(path, kCGPathElementCloseSubpath);
+    __closePath(path);
 }
 
 /**
- @Status Interoperable
+@Status Stub
+@Notes
 */
 CGRect CGPathGetBoundingBox(CGPathRef path) {
-    CGRect ret;
+    D2D1_RECT_F bounds;
 
-    if (path == NULL) {
-        return CGRectMake(std::numeric_limits<float>::infinity(), std::numeric_limits<float>::infinity(), 0, 0);
+    __closePath(path);
+
```

Comment:
A comment on the __closepath function has been added. It would be cumbersome to add a comment to every __closePath call.

---
In reply to: [83128915](https://github.com/Microsoft/WinObjC/pull/1138#discussion_r83128915) [](ancestors = 83128915)